packages/hardhat-core/src/internal/core/config/default-config.ts

```diff
@@ -62,63 +62,6 @@ export const defaultHardhatNetworkParams: Omit<
    * https://api-TESTNET.etherscan.io/api?module=block&action=getblocknobytime&timestamp=TIMESTAMP&closest=before&apikey=APIKEY
    */
   chains: new Map([
-    [
```

Comment:
Instead of using chain overrides, we can depend on EDR's chain configurations for the removed maps